### ISSUE TITLE
Tasks - Compact snippet issue. 

### ISSUE THREAD

**Issue by Drakemoor** [replete/obsidian-minimal-theme-css-snippets#7]
Tasks - Compact
When the text is long just enough to fill a line but shorter than to flow on the next line, hovering over the task's backlink has an issue: it's impossible to click on the backlink since it hides when one moves the mouse pointer.  

![Animation](https://github.com/replete/obsidian-minimal-theme-css-snippets/assets/93224920/b19d05bd-1eca-44e5-888c-f6d0f5cc4354)

Is there a solution for this? let me know if I need to provide more info.

**Comment by replete** [comment]
Looks like the CSS font-sizes etc are not compatible with your theme. The solution is tweaking the values in CSS yourself until it looks right. I'm afraid I don't offer support it as I'm no longer using it.

Moved this snippet into the 'No longer developed' category, and I've also handed off the icon font generator tool to the Obsidian Tasks Group at https://github.com/obsidian-tasks-group/obsidian-tasks-custom-icons - so this kind of feature could be built into Tasks one day... in the meantime I'm afraid you'll have to edit some CSS to get what you want.
